core/src/main/java/org/radargun/stages/cache/background/BackgroundOpsManager.java

```diff
@@ -397,14 +397,10 @@ private synchronized void stopBackgroundThreads(boolean stressors, boolean check
       // interrupt all threads
       log.debug("Stopping stressors");
       if (stressors && stressorThreads != null) {
-         for (int i = 0; i < stressorThreads.length; i++) {
-            stressorThreads[i].requestTerminate();
-         }
+         Stressor.requestTerminate();
```

Comment:
This does not fix the problem, it's just a nasty workaround, and breaks the case when you have multiple BackgroundOpsManagers.